<launch>
    <group ns="camera">
        <node pkg="nodelet" type="nodelet" name="color_detector" output="screen" args="load rr_iarrc/color_detector camera_nodelet_manager" required="true">
         <param name="load_file" type="string" value= "$(find rr_iarrc)/saved_hsv/example.txt"/>
         <param name="camera_image" type="string" value= "/camera_image_rect"/>
         <param name="detection_topic" type="string" value= "/hsv_detected"/>
         <param name="hsv_values_topic" value="/hsv_values"/>
        </node>
    </group>
</launch>
